### DISCUSSION TITLE
Vlad Diffusion by Olivio Sarikas Tutorial

### DISCUSSION THREAD

**Discussion by Centurion-Rome** [vladmandic/sdnext#426] [General]
Vlad Diffusion by Olivio Sarikas: Install Guide + Best Settings: https://www.youtube.com/watch?v=mtMZGdCjUwQ

Great, it's happens, I hope we get more devs moved here :)

**Also big thanks to devs that joined this fork!**

**Comment by puzzleme101** [comment]
can you please help me out in downloading temporal net in vlad diffusion
